advocacy_docs/supported-open-source/patroni/migrating.mdx

```diff
@@ -0,0 +1,323 @@
+    1. `etcd-node-1`.
+    2. `etcd-node-2`.
+    3. `etcd-node-3`.
+2. You have a Postgres cluster that needs to be migrated, which is composed of the following nodes:
+    1. `postgres-node-1` &mdash; the primary node.
+    2. `postgres-node-2` &mdash; a standby of `postgres-node-1`.
+    3. `postgres-node-3` &mdash; a standby of `postgres-node-1`.
+3. The existing Postgres cluster uses replication slots to control replication from primary to standbys, and they are named:
+    1. `slot_for_postgres_node_2` &mdash; slot used by node `postgres-node-2`.
+    2. `slot_for_postgres_node_3` &mdash; slot used by node `postgres-node-3`.
+4. You already have a few Postgres users in the cluster:
+    1. `postgres` with password `postgres_password` &mdash; Postgres superuser.
+    2. `replicator` with password `replicator_password` &mdash; used by replication connections.
+    3. `rewind` with password `rewind_password` &mdash; user with rights for executing `pg_rewind`.
+5. You are running Postgres 15 with the following characteristics:
+    1. Postgres is running on port `5432`.
+    2. Postgres data directory is `/var/lib/pgsql/15/data`.
+    3. Postgres binaries directory is `/usr/pgsql-15/bin`.
+6. You already have installed Patroni in the Postgres nodes.
+7. The systemd unit names are:
+    1. `postgres` &mdash; for Postgres.
+    2. `patroni` &mdash; for Patroni.
+    3. `edb-efm` &mdash; in case EFM is being used in the existing cluster.
+    4. `repmgrd` &mdash; in case repmgr is being used in the existing cluster.
+8. The configuration files are:
+    1. `/etc/patroni.yml` &mdash; for Patroni.
+    2. `/etc/repmgr.conf` &mdash; in case repmgr is being used in the existing cluster.
+9. The cluster name is `test_cluster`. This is relevant only for EFM and Patroni.
+
+**Notes:**
+
+* Please adapt the example names according to your actual cluster.
+* Setting up Watchdog and HAProxy are out of the scope of this example. However, as they are part of the recommended architecture, we recommend you read the following documentation:
+    * About Watchdog:
+        * [Quick start on RHEL 8 - 3. Watchdog](rhel8_quick_start#3-watchdog)/[Quick start on Debian 11 - 3. Watchdog](debian11_quick_start#3-watchdog) &mdash; set up an Watchdog device.
+        * [Quick start on RHEL 8 - 4. Patroni](rhel8_quick_start#4-patroni)/[Quick start on Debian 11 - 4. Patroni](debian11_quick_start#4-patroni) &mdash; configure Patroni to use Watchdog.
+    * About HAProxy:
+        * [Quick start on RHEL 8 - 5. HAProxy](rhel8_quick_start#5-haproxy)/[Quick start on Debian 11 - 5. HAProxy](debian11_quick_start#5-haproxy) &mdash; set up HAProxy.
+* This example uses Postgres 15. If you are running EDB Postgres Extended or EDB Postgres Advanced Server, then we recommend you also read the following documentation:
```

Comment:
Postgres -> PostgreSQL
Is it really using PG 15? Isn't postgresql-15 the unit name by default on RHEL?

Reply:
Yes, I just assumed `postgres` unit to make it simpler. Would you have any suggestion to modify PGDATA and bin dir path so we can get out of mentioning "PostgreSQL 15"?

Reply:
I mean, I would like to avoid using platform specific stuff in this page.